addons/mail_group/views/portal_templates.xml

```diff
@@ -273,50 +197,122 @@
                 <button class="btn btn-link o_mg_read_more"
                     t-attf-data-href="/groups/#{slug(group)}/#{slug(parent_message)}/get_replies"
                     t-attf-data-last-displayed-id="#{messages[-1].id}">
-                    show <t t-esc="msg_more_count"/> more replies
+                    <t t-esc="msg_more_count"/> more replies
                 </button>
             </p>
         </div>
     </template>
 
+    <template id="message_attachments">
+        <div class="o_mg_link_parent">
+            <t t-set="attachments" t-value="message.sudo().attachment_ids"/>
```

Comment:
Won't that make a sudo on each message, leading to bad performances ? Moreover I think templates should not sudo things. Controllers should give data so that there is as few logic as possible in templates. Probably linked to a sudo you removed in a controller, not sure why ...

Reply:
I can remove this and give the messages SUDOed

Reply:
In the end I can move the sudo on the controller side without issue, I did it